django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
     """
     Converts an aware datetime.datetime to local time.
 
     Local time is defined by the current time zone, unless another time zone
     is specified.
     """
```

Comment:
If it's changed to return naive current local time when `USE_TZ=False` (as discussed above) that should also be mentioned here.